tests/sentry/auth_v2/utils/test_session.py

```diff
@@ -11,32 +12,34 @@
 
 
 class MockSession(SessionBase):
```

Comment:
is there a reason to use a mock here?  the actual django session objects are pretty easy to construct